RHDEVDOCS-6210: Remove deprecated ClusterTask from docs (#84204)
openshift/openshift-docs: modules/op-resolver-cluster-tasks-ref.adoc

An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.

> Until version 1.16, {pipelines-shortname} included `ClusterTask` functionality. Versions 1.17 and later no longer include this functionality. If your pipelines use `ClusterTask` references, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the previously existing `ClusterTask` definitions.

Reviewer: Is it `you can re-create them with the tasks` or should we say `you can update them with the tasks`

Reviewer 2: About "update them" - I am really not sure how you can update a pipeline in place? If you *can* do that, let's make it "update or re-create them"

About "renaming the file" - I don't think so. it's actually "op-resolver-cluster", the name of the cluster resolver file, plus "-tasks-ref".

Reviewer: > About "renaming the file" - I don't think so. it's actually "op-resolver-cluster", the name of the cluster resolver file, plus "-tasks-ref".

> Until version 1.16, {pipelines-shortname} included `ClusterTask` functionality. Versions 1.17 and later no longer include this functionality. If your pipelines use `ClusterTask` references, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the previously existing `ClusterTask` definitions.

Reviewer 3: ```suggestion
Until version 1.16, {pipelines-shortname} included `ClusterTask` functionality. Versions 1.17 and later do not include this functionality. If your pipelines use `ClusterTask` references, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the previously existing `ClusterTask` definitions.
```
I would simplify this at some point for minimalism. It _sounds like_ 1.17 and later used to also have this function, but it was removed ("no longer"). If they never had it, I would just say they don't. Not required for merge, just a thought.

An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.

> `ClusterTask` functionality is deprecated since {pipelines-shortname} 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.

Author: I am putting it back for now before deciding what to do next :+1:

> `ClusterTask` functionality is deprecated since {pipelines-shortname} 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.

Reviewer 2: Suggestion:

`Until version 1.16, {pipelines-shortname} included `ClusterTask` functionality. Versions 1.17 and later no longer include this functionality. If your pipelines use `ClusterTask` references, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the previously existing `ClusterTask` definitions.
`

> `ClusterTask` functionality is deprecated since {pipelines-shortname} 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.

Reviewer 2: Note that the use of `ClusterTasks` is not compliant with the current OpenShift documentation standards/ As we are changing this paragraph we must make it compliant or it will fail peer/merge review.

> `ClusterTask` functionality is deprecated since {pipelines-shortname} 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.

Author: I implemented the changes as suggested. I really like that solution.
